tag-emeritus-leaders.md

```diff
@@ -0,0 +1,15 @@
+# TAG Security Chair Emeriti
```

Comment:
This file feels out of place at the top level of the repository. A better location would be under the community directory, specifically under community/assets in the revised IA. The gratitude statement added to the repo would probably fit well in this file

Reply:
Agree.
Can you please clarify? There is no "assets" folder under community. Should I create the sub-folder?

I added changes as I understood. Happy to update with further guidance.

Reply:
This is work underway in another open PR. Since the rest of the leadership names were removed, probably this should be left out for now and revisited later.